packages/next-templates/src/components/PreFooterNav/PreFooterNav.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link, BreadcrumbNavSeparator } from '@utrecht/component-library-react';
+import { IconChevronRight, IconChevronUp } from '@tabler/icons-react';
+import './pre-footer-nav.css';
+
+export const PreFooterNav = () => (
+  <div className="voorbeeld-pre-footer">
+    <Link className="voorbeeld-pre-footer-link" href="/404">
+      {/* empty link - page doesnt exist yet */}
+      <IconChevronRight />
+      Wat vindt u van deze pagina
+    </Link>
+    <Link className="voorbeeld-pre-footer-link" href="/404">
```

Comment:
hier wil je waarschijnlijk dit doen https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_an_element_on_the_same_page en het dan in een property zetten

Reply:
Hai! Ik heb net deze manier toegepast op de pagina en heb toch nog wat vragen.

Als ik het id op Heading1 zet, is het resultaat niet echt de top van de pagina. Dit is waar je naartoe gestuurd wordt:
<img width="1259" alt="Scherm­afbeelding 2024-04-10 om 13 09 13" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/12d258c9-3d1d-41d0-b430-51de4d928869">

En dit is wat ik zou willen zien:
<img width="1451" alt="Scherm­afbeelding 2024-04-10 om 13 25 44" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/f6da4de2-4ef9-4885-ad1d-206140b42079">


Is Heading1 niet het juiste element hiervoor? (leek mij wel - anders zit je in de breadcrumb of header?) Of is er iets wat ik anders kan doen? Op de website van Utrecht gaat de "naar boven" link wel echt naar boven maar zij hebben dat niet op deze manier gedaan voor zover ik kan zien. 

Ook komt er nu in de URL #naar-boven te staan. Ik weet niet zeker of:
1. dit de bedoeling is?
2. dit beter is dan een lege href of #?
3. het id beter een andere naam kan krijgen? (bijv. iets rondom het klachtenformulier?)

Reply:
zet de id op de skip link dan gaat het helemaal naar boven (high probability). En screen readers gaan direct weer naar de skip link zodat ze zelf kunnen kiezen waar ze heen moeten gaan.

Reply:
Dit leek een super goed idee, werkte alleen helaas niet omdat de pagina niet naar boven ging maar alleen de skip link visible maakte bovenaan de viewport die op dat moment open stond. 

Volgende poging: het logo in de header.

Reply:
Logo in header brengt de pagina wel naar boven maar met focus op het logo en ook nét niet helemaal bovenaan.

Reply:
### Gekozen oplossing

Uiteindelijk heb ik het id op het HTML element gezet in layout.tsx.
Als het goed is kunnen andere pagina's met een "naar boven" link hier dus ook gebruik van maken.
<img width="528" alt="Scherm­afbeelding 2024-04-17 om 10 09 11" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/cbb511b1-9ccd-40fa-832c-a041a4683150">

### Visueel

Na het klikken op de link, ben je weer helemaal bovenaan de pagina (wat op skiplink en logo in header niet het geval was). Na het klikken komt de URL er zo uit te zien:
<img width="470" alt="Scherm­afbeelding 2024-04-17 om 10 09 32" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/1a73c16c-2dea-44f6-8c00-a6669f5b0699">

### Tab

De link is bereikbaar met tab en werkt zoals verwacht. Het eerstvolgende tabbare element na de "naar boven" link is de "skip to content" link.

### Screen reader

Bij screen reader gebruik wordt niet benoemd waar de gebruiker zich bevindt na het klikken op de "naar boven" link. Hier heb ik een [issue](https://github.com/frameless/gemeentevoorbeeld.nl/issues/453) voor aangemaakt. 

Ook blijft er een omlijning van de link achter halverwege de pagina wanneer je met de screen reader en toetsenbord de link gebruikt. 
<img width="321" alt="Scherm­afbeelding 2024-04-17 om 10 16 37" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/f196bc3d-323a-4abd-9547-a498062cae75">

Reply:
`id="top-of-the-page-link"` is niet nodig, HTML heeft hier al een oplossing voor: `href="#"`

https://html.spec.whatwg.org/multipage/browsing-the-web.html#select-the-indicated-part

> If fragment is the empty string, then return the special value top of the document.

en:

> Otherwise, if document's indicated part is top of the document, then: [...] Scroll to the beginning of the document for document. [CSSOMVIEW]

Reply:
Nice, dan is de cirkel nu rond hahah. Ik snapte al niet waarom dit geen optie was. 🤔 
Code is aangepast!